<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZPMainFrame</class>
 <widget class="QMainWindow" name="ZPMainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZPMainFrame</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit_status_net"/>
    </item>
    <item>
     <widget class="QTextEdit" name="DebugTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Control">
    <property name="title">
     <string>&amp;Control</string>
    </property>
    <addaction name="action_Start_Stop"/>
    <addaction name="actionReload_config_file"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_Control"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Start_Stop"/>
   <addaction name="actionReload_config_file"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
   <addaction name="actionStop_Debug"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Start_Stop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Resources/cn11.png</normaloff>
     <normalon>:/icons/Resources/cn6.png</normalon>:/icons/Resources/cn11.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start/Stop</string>
   </property>
   <property name="toolTip">
    <string>Start or stop the server</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Resources/4013.png</normaloff>:/icons/Resources/4013.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionReload_config_file">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Resources/Color, Lime.png</normaloff>:/icons/Resources/Color, Lime.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reload config file</string>
   </property>
   <property name="toolTip">
    <string>Reload Config file</string>
   </property>
  </action>
  <action name="actionShow_Window">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Resources/Color, Bondi.png</normaloff>:/icons/Resources/Color, Bondi.png</iconset>
   </property>
   <property name="text">
    <string>Show&amp;Window</string>
   </property>
   <property name="toolTip">
    <string>Show main window</string>
   </property>
  </action>
  <action name="actionStop_Debug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="debug" resource="resource.qrc">
     <normaloff>:/icons/Resources/hanukkah_03.png</normaloff>:/icons/Resources/hanukkah_03.png</iconset>
   </property>
   <property name="text">
    <string>Stop&amp;Debug</string>
   </property>
   <property name="toolTip">
    <string>Stop Debug</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
